config/routes.rb

```diff
@@ -8,7 +8,7 @@
   end
   resources :answers
   resources :weekly_friends, only: [:show], controller: "weekly_friends"
-  post "/auth/login", to: "authentication#login"
-  get "/weekly_question", to: "week_question#show"
+  post "/auth/login", to: "authentication#login", as: "login"
```

Comment:
```suggestion
  post "/auth/login", to: "authentication#login", as: :login
```
It's common to use atoms for things that are identifiers, such as hash keys and route names